<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>160</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="operationModeLayout">
     <item>
      <widget class="QLabel" name="operationModeLabel">
       <property name="text">
        <string>Operation Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="BioAmp8">
       <property name="text">
        <string>Single BioAmp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="BioAmp16">
       <property name="text">
        <string>Cascade Mode</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="sinusoidTestButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>sin.png</normaloff>sin.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>281</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>184</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="availablePortsLayout">
     <item>
      <widget class="QLabel" name="availablePortsLabel">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="availablePorts">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>30</height>
          </size>
         </property>
         <property name="currentText">
          <string extracomment="Select Port">No port selected</string>
         </property>
         <property name="modelColumn">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>No port selected</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="portStatusLabel">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>verticalLayoutWidget</zorder>
   <zorder></zorder>
   <zorder>availablePorts</zorder>
   <zorder>availablePortsLabel</zorder>
   <zorder>label</zorder>
   <zorder>availablePorts</zorder>
   <zorder>portStatusLabel</zorder>
   <zorder>availablePorts</zorder>
   <zorder>sinusoidTestButton</zorder>
   <zorder>infoLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>563</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
